packages/teams-js/src/internal/telemetry.ts

```diff
@@ -1,4 +1,4 @@
-import { debug as registerLogger, Debugger } from 'debug';
+import { debug as registerLogger, Debugger } from 'debug/src/browser';
```

Comment:
>'debug/src/browser'; [](http://example.com/codeflow?start=50&length=20)

Can you leave a comment here explaining to any naive engineer reading this code why we aren't using the root level package?